scripts/browserstack.py

```diff
@@ -254,6 +263,60 @@ def confirm(message):
         answer = input(message + " [Y/N]? ").lower()
     return answer == "y"
 
+def runTests(appUrl, testUrl, isNightly, testClasses):
+   print("RUNNING " + str(len(testClasses)) + " test cases")
+   buildId = executeTests(appUrl, testUrl, isNightly, testClasses)
+   exitStatus = 1
+   if(buildId != None):
```

Comment:
There are some minor formatting inconsistencies. But I don't think we care too much.

Reply:
When I run my auto-formatter on this script, basically every line is impacted. I'd like to just reformat the whole thing, but it makes the diff impossible to read. So I can do that in a follow up PR